src/proto/streams/state.rs

```diff
@@ -362,7 +362,8 @@ impl State {
 
     pub fn is_remote_reset(&self) -> bool {
         match self.inner {
-            Closed(Cause::Error(ref e)) => e.is_local(),
+            // SEAN: WAT
```

Comment:
left over?

Reply:
lol yes

Reply:
probably, but it adds flavor to the code :joy: